distribution/docs/src/main/resources/_contents/_sources/federated-sources-contents.adoc

```diff
@@ -0,0 +1,11 @@
+
+=== Included Federated Sources
+
+The following federated sources are included with a standard installation of ${branding}:
+
+<<_csw_source,CSW Source>>:: The CSW source supports the ability to search collections of descriptive information (metadata) for data, services, and related information objects.
+<<_gmd_csw_source,GMD CSW Source>>:: _implements https://portal.opengeospatial.org/files/?artifact_id=6495[Application Profile ISO 19115/ISO19119]_. The GMD CSW source supports the ability to search collections of descriptive information (metadata) for data, services, and related information objects.
+<<_opensearch_source,OpenSearch Source>>:: The OpenSearch source provides a <<_federated_source,Federated Source>> that has the capability to do http://www.opensearch.org/Home[OpenSearch] queries for metadata from Content Discovery and Retrieval (CDR) Search V1.1 compliant sources.
+<<_twitter_source,Twitter Source>>:: The Twitter Source is an *EXAMPLE* source provided in the ${branding} SDK. It enables queries using the http://twitter.com/[Twitter API].
```

Comment:
The Twitter source is listed twice here, and neither one links to the Twitter Source Section.
![image](https://cloud.githubusercontent.com/assets/8041246/18290654/3613b4f8-7439-11e6-9941-3db889ca14f6.png)